configs/next/packages/liburcu/stage_1

```diff
@@ -105,7 +109,7 @@ atcfg_configure ()
 	PATH=${at_dest}/bin:${PATH} \
 	CC="${configure_cc}" \
 	CFLAGS="-g -O2 -fpic -m${AT_BIT_SIZE}" \
-	${ATSRC_PACKAGE_WORK}/configure \
+	./configure \
```

Comment:
Most of the ```stage_X``` files currently use ```${ATSRC_PACKAGE_WORK}/configure```.  Do you have reason to make this change?  If so, would it be useful (easy) to add a comment about it?

Reply:
There is no configure file in the sources of liburcu when getting the code from the repository, it is generated in the build directory by running the bootstrap script.

Reply:
OK.  Thanks for the info.